examples/openid.php

```diff
@@ -68,10 +66,8 @@ function Show_page($contents)
  * Display error and exit
  *
  * @param Exception $e Exception object
- *
- * @return void
  */
-function Die_error($e)
+function Die_error($e): void
```

Comment:
```suggestion
function Die_error(\Exception $e): void
```
I know it's out of the scope of this PR but it could be changed too as example files do not change often

Reply:
I will open up more PRs like this with these changes.